### PR TITLE
(maint) Prepare for `0.3.6` release

### PR DESCRIPTION
**Pull request by MikaelSmith**
Add entry for pod2man to README
Update changelog.
Bump project version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.3.6] - 2016-02-05
+
+### Fixed
+
+- Added version to Leatherman CMake config, so downstream projects can depend on a particular version.
+- Added pod2man macro for generating man pages.
```
Comment (branan) [inline CHANGELOG.md]: Should be in an `Added` section?

**Comment by whopper** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (MikaelSmith) [inline CHANGELOG.md]: Ah, yup.

**Comment by MikaelSmith** [comment]
Updated and passed CI.

**Pull request merged**